pkg/collector/corechecks/cluster/ksm/kubernetes_state_defaults.go

```diff
@@ -118,7 +118,8 @@ var (
 
 	// deniedMetrics used to configure the KSM store to ignore these metrics by KSM engine
 	deniedMetrics = options.MetricSet{
-		".*_created":                                       {},
+		// deny all *_created metrics except for kube_node_created
+		"^_created|^(?:[^k]|k[^u]|ku[^b]|kub[^e]|kube[^_]|kube_[^n]|kube_n[^o]|kube_no[^d]|kube_nod[^e]|kube_node.).*_created": {},
```

Comment:
This regex is still not perfect since, for example, it does **not** match `kube_created` whereas it should. Indeed, it’s of the form `*_created` and it’s not `kube_node_created`.

We can either:
* fix it and make it more unreadable or
* consider it’s good enough since it’s validated against the real cases in `TestCreationMetricsFiltering`.

Reply:
yes, let's not make it more unreadable 😅